wAbout/Logging-In.md

```diff
@@ -1,42 +1,88 @@
 ---
 title: Logging In
 layout: custom
-keywords: [login, ribbon]
-description: Logging in is simple and fast! After logging in once, you can set up auto-login, which saves you from entering credentials each time. You can also log into multiple accounts. 
+keywords: [login, ribbon, logout, new login, credentials]
+description: Logging in is simple and fast! Interject will remember your login and sign you in automatically subsequent visits.You can also add multiple logins and switch accounts easily.
 ---
 
-##  Before Using Interject, Here's How to Login 
+## Overview 
```

Comment:
That's a good point. My first thought would be that they should assume it's always the latest. Would there ever be a use case for showing pictures from different versions?

Reply:
Yea I have thought about this but I am not sure what the best answer is. There are many other pics throughout the site that simply use the latest version and may look different than previous versions. I think the user should assume it's the latest. But in regards to this login page, it might be beneficial to add a 2.3.34 version section since it is still popular.